DependencyInjection/SonataFormatterExtension.php

```diff
@@ -56,25 +56,29 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('ckeditor.xml');
         }
 
-        // TODO: To be removed when major version is changed
-        if (!isset($config['default_formatter'])) {
-            reset($config['formatters']);
-            $config['default_formatter'] = key($config['formatters']);
-        }
-
         if (!array_key_exists($config['default_formatter'], $config['formatters'])) {
-            throw new \InvalidArgumentException(sprintf('SonataFormatterBundle - Invalid default formatter : %s, available : %s', $config['default_formatter'], json_encode(array_keys($config['formatters']))));
+            throw new \InvalidArgumentException(sprintf(
```

Comment:
Personally, in this case I'll go as far as to prefer discrepancy, so as long as someone does not point me at some docs showing this is part of the sf coding standard, which I think is full of very questionable decisions like Yoda conditions or non-multiline-signatures that are 500 chars long (not kidding, there are things like this in sf), then I'll keep using and asking for linebroken exceptions. It gives better diffs, and people can actually read the exception messages that way.
I don't know why people are so afraid of hitting the carriage return key.